arch/arc/core/mpu/arc_mpu_v3_internal.h

```diff
@@ -0,0 +1,677 @@
+/*
+ * Copyright (c) 2019 Synopsys.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_ARCH_ARC_CORE_MPU_ARC_MPU_V2_INTERNAL_H_
```

Comment:
should be _V3_ not V2 (next line as well)

Reply:
done